scripts/update_docs.sh

```diff
@@ -21,9 +21,13 @@ cd "$(dirname "$(dirname "${SELF}")")"
 
 SITE_BASE="$(pwd)"
 
-# grab and unpack the latest matrix-docs build from buildkite
-rm -rf assets.tar.gz assets
-scripts/fetch-buildkite-artifact matrix-dot-org matrix-doc assets.tar.gz
+if [ -n "$BUILDKITE" ]; then
```

Comment:
I checked the documentation before applying your suggestion, @richvdh, but [Buildkite doesn’t seem to like that variable](https://buildkite.com/matrix-dot-org/matrix-dot-org/builds/3266#bbbeddd9-0eea-47c5-919d-d0ca5722a18f): it claims it is “unbound”, even though the “Environment” tab shows it first in the list. Any clue what’s going on?

Reply:
Updated the test, now it runs again both on BK and GHA.